src/fw/system/bootbits.c

```diff
@@ -102,32 +102,37 @@ uint32_t boot_version_read(void) {
 
 #elif MICRO_FAMILY_SF32LB52
 
-// TODO(SF32LB52): implement
-
-static uint32_t _bootbits;
-
 void boot_bit_init(void) {
-  _bootbits = BOOT_BIT_INITIALIZED;
+  rtc_init();
```

Comment:
you'll be calling rtc_init() twice now? is rtc_init prepared?

Reply:
removed rtc_init, not required in here.